systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/MultipleListenersST.java

```diff
@@ -0,0 +1,353 @@
+import io.strimzi.systemtest.resources.crd.KafkaTopicResource;
+import io.strimzi.systemtest.resources.crd.KafkaUserResource;
+import io.strimzi.systemtest.utils.kafkaUtils.KafkaTopicUtils;
+import io.strimzi.systemtest.utils.kafkaUtils.KafkaUserUtils;
+import org.apache.kafka.common.security.auth.SecurityProtocol;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static io.strimzi.systemtest.Constants.ACCEPTANCE;
+import static io.strimzi.systemtest.Constants.EXTERNAL_CLIENTS_USED;
+import static io.strimzi.systemtest.Constants.INTERNAL_CLIENTS_USED;
+import static io.strimzi.systemtest.Constants.LOADBALANCER_SUPPORTED;
+import static io.strimzi.systemtest.Constants.NODEPORT_SUPPORTED;
+import static io.strimzi.systemtest.Constants.REGRESSION;
+
+@Tag(REGRESSION)
+public class MultipleListenersST extends AbstractST {
+
+    private static final Logger LOGGER = LogManager.getLogger(MultipleListenersST.class);
+    public static final String NAMESPACE = "multi-listener-namespace";
+
+    // only 4 type of listeners
+    private Map<KafkaListenerType, List<GenericKafkaListener>> testCases = new HashMap<>(4);
+
+    @Tag(NODEPORT_SUPPORTED)
+    @Tag(EXTERNAL_CLIENTS_USED)
+    @Test
+    void testMultipleNodePorts() {
+        runListenersTest(testCases.get(KafkaListenerType.NODEPORT));
+    }
+
+    @Tag(INTERNAL_CLIENTS_USED)
+    @Test
+    void testMultipleInternal() {
+        runListenersTest(testCases.get(KafkaListenerType.INTERNAL));
+    }
+
+    @Tag(NODEPORT_SUPPORTED)
+    @Tag(EXTERNAL_CLIENTS_USED)
+    @Tag(INTERNAL_CLIENTS_USED)
+    @Tag(ACCEPTANCE)
+    @Test
+    void testCombinationOfInternalAndExternalListeners() {
+        List<GenericKafkaListener> multipleDifferentListeners = new ArrayList<>();
+
+        List<GenericKafkaListener> internalListeners = testCases.get(KafkaListenerType.INTERNAL);
+        List<GenericKafkaListener> nodeportListeners = testCases.get(KafkaListenerType.NODEPORT);
+
+        multipleDifferentListeners.addAll(internalListeners);
+        multipleDifferentListeners.addAll(nodeportListeners);
+
+        // run INTERNAL + NODEPORT listeners
+        runListenersTest(multipleDifferentListeners);
+    }
+
+    @Tag(LOADBALANCER_SUPPORTED)
+    @Tag(EXTERNAL_CLIENTS_USED)
+    @Test
+    void testMultipleLoadBalancers() {
+        runListenersTest(testCases.get(KafkaListenerType.LOADBALANCER));
+    }
+
+    @OpenShiftOnly
+    @Tag(EXTERNAL_CLIENTS_USED)
+    @Test
+    void testMultipleRoutes() {
+        runListenersTest(testCases.get(KafkaListenerType.ROUTE));
+    }
+
+    @Tag(NODEPORT_SUPPORTED)
+    @OpenShiftOnly
```

Comment:
Probably best to have a consistent order for the annotations, and not split annotations of the same type.